src/components/popover/bl-popover.stories.mdx

```diff
@@ -0,0 +1,301 @@
+         buttonText:'Custom Background'}}
+         play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+</Canvas>
+
+
+## Border Color
+You can give `--bl-popover-border-color` for popover border color. Default color is `#ef6114`.
+
+<Canvas>
+  <Story name="Border Color"
+         args={{  id:'5', style: '--bl-popover-border-color:blue;', buttonText:'Border Color' }}
+         play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+</Canvas>
+
+## Padding
+You can give `--bl-popover-padding` for popover padding. Default padding is `1rem`.
+
+<Canvas>
+  <Story name="Padding"
+         args={{  id:'6', style: '--bl-popover-padding:2rem;', buttonText:'Large Padding' }}
+         play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+  <Story name="Padding 0"
+         args={{  id:'7', style: '--bl-popover-padding:0;', buttonText:'No Padding' }}
+         play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+</Canvas>
+
+## Border Radius
+You can give `--bl-popover-border-radius` for border radius. Default padding is `0.25rem`.
+
+<Canvas>
+  <Story name="Border Radius"
+         args={{  id:'8', style: '--bl-popover-border-radius:0;', buttonText:'Border Radius 0' }}
+         play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+</Canvas>
+
+## Offset
+You can give `offset` for padding from target. Default padding is `10`.
+
+<Canvas>
+  <Story name="Default Offset"
+         args={{  id:'9',placement:'right', buttonText:'Default Offset'}}
+         play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+  <Story name="Custom Offset"
+         args={{  id:'10',placement:'right', offset:'40', buttonText:'Custom Offset'}}
+         play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+</Canvas>
+
+## FitSize
+You can give `fitSize` for auto size. Default fitSize is `false`.
+
+<Canvas>
+  <Story name="Fitsize"
+         args={{  id:'11', fitSize:'true', buttonText:'Long content text for fitSize'}}
+         play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+</Canvas>
+
+
+
+## Shift
+You can give `shift` for floating element along the specified axes in order to keep it in view. Default fitSize is `false`.
+
+<Canvas>
+  <Story name="Shift"
+         args={{  id:'12', shift:'true' }}
+         play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+</Canvas>
+
+
+## Placement
+
+You can give `placement` for popover direction.The position of the popover and arrow change according to this property.  Default placement is `top`.
+
+You can use the other options:
+
+* `top-start`
+* `top`
+* `top-end`
+* `bottom-start`
+* `bottom`
+* `bottom-end`
+* `left-start`
+* `left`
+* `left-end`
+* `right-start`
+* `right`
+* `right-end`
+
+If there is not enough room for the popover in the given placement, the popover is repositioned automatically according to the empty area.
+
+For example, if there is not enough room on the top, the popover is shown on the bottom.
```

Comment:
This relies on `shift` parameter. Maybe you can move this example to `shift` title.